nilearn/image/image.py

```diff
@@ -477,47 +493,48 @@ def _compute_mean(imgs, target_affine=None,
 
 def mean_img(imgs, target_affine=None, target_shape=None,
              verbose=0, n_jobs=1):
-    """ Compute the mean of the images (in the time dimension of 4th dimension)
+    """Compute the mean of the images over time.
```

Comment:
```suggestion
    """Compute the mean of the images over time or the 4th dimension.
```
It may not be clear for everyone that the 4th dim is time.